pkg/operator/sync.go

```diff
@@ -341,26 +343,91 @@ func (optr *Operator) syncRenderConfig(_ *renderConfig) error {
 		caData[CA.File] = string(CA.Data)
 	}
 
-	err = optr.kubeClient.CoreV1().ConfigMaps("openshift-config-managed").Delete(context.TODO(), "merged-trusted-image-registry-ca", metav1.DeleteOptions{})
+	cm, err = optr.kubeClient.CoreV1().ConfigMaps("openshift-config-managed").Get(context.TODO(), "merged-trusted-image-registry-ca", metav1.GetOptions{})
```

Comment:
Should we try to read this off a lister instead of doing a Get directly via API? (I guess I'm not sure if we even have a lister for this currently).